App.js

```diff
@@ -6,6 +6,7 @@ import Wizard from "./composites/OnboardingWizard/OnboardingWizard";
 import Config from "./composites/OnboardingWizard/config/production-config";
 import SearchResultsEditor from "./composites/SearchResultEditor/SearchResultEditor";
 import SnippetPreview from "./composites/SnippetPreview/components/SnippetPreview";
+import ProgressBarPreview from "./composites/SnippetPreview/components/ProgressBarPreview";
```

Comment:
Please don't commit ProgressbarPreview, because that is essentially an acceptance test. If we would commit a preview for every component, we would soon have a gazillion tabs/button in the app. Please provide test instructions in the PR message on how to test the ProgressBar in `components/SnippetPreview.js` instead. If necessary, you can include a code snippet that the tester can copy and paste.